packages/@expo/cli/src/start/resolveOptions.ts

```diff
@@ -35,9 +38,12 @@ export async function resolveOptionsAsync(projectRoot: string, args: any): Promi
     tunnel: args['--tunnel'],
   });
 
+  // TODO: Add a third option which is auto detecting if the user is using `expo-dev-client` or `expo-dev-launcher`.
```

Comment:
the third option being native app without expo-dev-client?

(also, people shouldn't use expo-dev-launcher directly, should only ever be installed via expo-dev-client)

Reply:
Third option is defining nothing and us inferring the type based on if expo-dev-client is installed.